incontext

```diff
@@ -39,12 +47,35 @@ verbose = '--verbose' in sys.argv[1:] or '-v' in sys.argv[1:]
 logging.basicConfig(level=logging.DEBUG if verbose else logging.INFO, format="[%(levelname)s] %(message)s")
 
 
+class NotFound(Exception):
+    pass
+
+
+class Docker(object):
```

Comment:
This should also support adding volume mounts and the functionality for correctly determining the relevant set of volume mounts should be added in here.

Reply:
Won't fix right now.